index.js

```diff
@@ -23,28 +33,41 @@ module.exports = bundler => {
 			return;
 		}
 		console.log('');
+
 		const spinner = ora(chalk.grey('Fixing og:image link')).start();
 		const start = Date.now();
 
 		const htmlPath = path.join(bundler.options.outDir, 'index.html');
 		const html = fs.readFileSync(htmlPath).toString();
+		let hasErrors = false;
 
-		const ogImageTag = getMetaTag(html, 'og:image');
-		const ogImageContent = getMetaTagContent(ogImageTag);
+		try {
+			const ogImageTag = getMetaTag(html, 'og:image');
+			const ogImageContent = getMetaTagContent(ogImageTag, 'og:image');
 
-		const ogUrlTag = getMetaTag(html, 'og:url');
-		const ogUrlContent = getMetaTagContent(ogUrlTag);
+			const ogUrlTag = getMetaTag(html, 'og:url');
+			const ogUrlContent = getMetaTagContent(ogUrlTag, 'og:url');
 
-		const absoluteOgImageUrl = url.resolve(ogUrlContent, ogImageContent);
-		const ogImageTagAbsoluteUrl = ogImageTag.replace(ogImageContent, absoluteOgImageUrl);
-		const patchedHtml = html.replace(ogImageTag, ogImageTagAbsoluteUrl);
+			const absoluteOgImageUrl = url.resolve(ogUrlContent, ogImageContent);
+			const ogImageTagAbsoluteUrl = ogImageTag.replace(ogImageContent, absoluteOgImageUrl);
+			const patchedHtml = html.replace(ogImageTag, ogImageTagAbsoluteUrl);
 
-		fs.writeFileSync(htmlPath, patchedHtml);
+			fs.writeFileSync(htmlPath, patchedHtml);
+		} catch (error) {
+			spinner.fail(error.message);
+
+			hasErrors = true;
+		}
 
 		const end = Date.now();
+		const symbol = hasErrors ? chalk.red('✖') : '✨ ';
```

Comment:
Do we need a space in `'✖'` here to keep the style consistent with the `'✨ '` symbol?

Reply:
In this case it looks better without a space, as `spinner.fail` does not have a space either:

<img width="354" alt="Screenshot 2020-03-19 at 17 20 38" src="https://user-images.githubusercontent.com/10677263/77089567-45b41800-6a06-11ea-960c-5c56f796735a.png">